sdk/go/auto/errors_test.go

```diff
@@ -416,7 +426,11 @@ func TestRuntimeErrorPython(t *testing.T) {
 	}
 
 	// install Pulumi Python SDK from the current source tree, -e means no-copy, ref directly
-	pyCmd := python.VirtualEnvCommand(filepath.Join(pDir, "venv"), "python", "-m", "pip", "install", "-e", pySDK)
+	pyCmd, err := tc.Command(context.Background(), "-m", "pip", "install", "-e", pySDK)
```

Comment:
I see `-m` being passed in to `tc.Command` as first argument everywhere.  Makes me wonder if we should just have `tc.Command` prepend that?  Not sure if there's any python toolchains out there that don't require that, but I could imagine someone e.g. wanting a "System" toolchain that doesn't rely on a venv at all, that would be made possible if we don't pass that as first argument everywhere?  Or is there another reason for this?

Reply:
The "system" toolchain can be used by not passing a `Virtualenv` argument. We don't pass `-m` when running the user's pulumi program for example. I think there might be some cases in the tests as well.

Maybe we can have the existing `toolchain.Command` and `toolchain.ModuleCommand` for when we use `-m` https://docs.python.org/3/using/cmdline.html#cmdoption-m

Reply:
Ah that makes sense.  I do think a `ModuleCommand` would be nice to have.

Reply:
I added `ModuleCommand` here https://github.com/pulumi/pulumi/pull/16243/commits/62de2b049261b96a2b2e91f5ced7d139debb2339